pkg/ruler/rulestore/objectclient/rule_store.go

```diff
@@ -8,15 +8,15 @@ import (
 	"io/ioutil"
 	"strings"
 
+	"github.com/cortexproject/cortex/pkg/chunk"
```

Comment:
Replace with the Loki one.

Reply:
good catch. pushed a few commits changing it, wdyt?